ske-operator/values.yaml

```
@@ -69,3 +69,10 @@ skeDeployment:
   enabled: true
   # The version to deploy
   version: "latest"
+  tlsConfig:
+    certManagerConfig:
+      disabled: false
+    # if certManagerConfig.disabled is set to false, the following values are required
+    # skeDeployment.tlsConfig.webhookCACert:
+    # skeDeployment.tlsConfig.webhookTLSKey:
+    # skeDeployment.tlsConfig.webhookTLSCert:
```

```suggestion
    # if certManagerConfig.disabled is set to false, the following values are required
    # webhookCACert:
    # webhookTLSKey:
    # webhookTLSCert:
```

These already live at `global.skeDeployment.tlsConfig` so this path is not needed. Same for the ske-operator values.

---

I felt the indentation of the comment wasn't making it 100% clear that it shouldn't be under `certManagerConfig` but instead a key peered with it, which is why I did this. Wdyt?